### DISCUSSION TITLE
Some questions regarding MMR calculations

### DISCUSSION THREAD

**Discussion by yrtnsky** [Heroes-Profile/heroesprofile#1002] [General]
In the match https://www.heroesprofile.com/Match/Single/53280025 just noticed somethingn strange - players with higher MMR got much higher addition to their MMR. But this is nonsense.
The player that spent most time on base got the highest increment and this player have the lowest Heroes Profile Rating.
Can we know how this is calculated?

**Comment by Zemill** [comment]
MMR is not based on any performance stat.  It is a win rate metric scaled against the other players in the game, and all others games you've played.

MMR is not infinitely scaling metric, the point of an MMR system is to place you into a distribution representing your skill.  The closer and closer you get to what the system thinks your skill (MMR) is, the less and less it will increase or decrease your MMR, pretty much solidifying it.


I assume you are talking about the Qhira player as they received 9.45 MMR which is higher than the other players.  The closest player is the Raynor player.

If I just use account level as a basis as thats somewhat comparative to games played, the Qhira player has less games played than any of the other players.  So the MMR system is less sure of their MMR than the other players, so they have a higher swing.  Lets use the player playing The Lost Vikings.  If it was just a performance metric, you would assume because the Qhira player had the highest increase in player MMR, they would have the highest for Hero MMR.  We can see that is not the case.  It shows that the Qhira player has played Qhira more than The Lost Vikings player has played The Lost Vikings.  So the Hero MMR system is more sure of the Qhira players Hero MMR on Qhira than it is of The Lost Vikings players Hero MMR on The Lost Vikings.


Let me know if you need more information.

**Reply by yrtnsky** [reply]
This statement just compromise the whole idea of this website's rating.
Also it does not take into account that players with time will play better.
Players with higher MMR should perform better than players with lower, so their expected value is lower.
If they played worse they should not get bigger increase.
In other words it looks like the most useless player of the team in the match got the biggest reward.

**Reply by Zemill** [reply]
"Played worse" is entirely subjective with no basis in reality.  If an Illidan player dies 10 times in a game but in doing so their team secures all objectives and bosses, is that considered bad play?  I doubt it, BUT, any system that only looks at the raw stat values is going to think that Illidan is feeding for no reason.  That is just one example.


Blizzard attempted to implement a ranked season where they took into account performance stats to increase or decrease players in the ranked season and realized quickly, within a week, that it was a terrible system, and took it out immediately.  All that a system focusing on stats does is make players play in sub-optimal ways to game the system to get the most possible points possible, which Blizzard learned quickly.

Feel free to research how other games/sites do MMR, and you will learn that in a vast majority of them, it is done in this way. 


We are open to an improved MMR system, so if you have some real solutions in mind, feel free to let us know
